doc/release-notes.md

```diff
@@ -55,6 +68,7 @@ debug the release candidates.
 
 These releases are considered obsolete. Old release notes can be found here:
 
+- [v22.1.1](https://github.com/dashpay/dash/blob/master/doc/release-notes/dash/release-notes-22.1.1.md) released Feb/17/2025
```

Comment:
You forgot to add the old release notes file to the commit it seems.

Reply:
resolved